app/assets/stylesheets/landing_page/css-index.css

```diff
@@ -0,0 +1,1035 @@
+/**

```

Comment:
All custom: Font sizes, colours, padding, margins should be pre-saved into variables, and used in styles definition. That way we'll have standardized, easy to modify styles.